extensions/ql-vscode/src/remote-queries/sample-data.ts

```diff
@@ -99,7 +99,23 @@ export const sampleRemoteQueryResult: RemoteQueryResult = {
 };
 
 
-const createAnalysisResults = (n: number) => Array(n).fill({ 'message': 'Sample text' });
+const createAnalysisResults = (n: number) => Array(n).fill(
+  {
+    message: 'This shell command depends on an uncontrolled [absolute path](1).',
+    severity: 'Error',
+    filePath: 'npm-packages/meteor-installer/config.js',
+    contextRegion: {
+      startLine: 253,
+      endLine: 257,
+      text: '  if (isWindows()) {\n    //set for the current session and beyond\n    child_process.execSync(`setx path "${meteorPath}/;%path%`);\n    return;\n  }\n',
+    },
+    codeRegion: {
```

Comment:
(Ran out of time for a proper review today, but here's a small comment before I forget 💭 )

Minor: I'm still getting a bit confused about what "codeRegion" means 😅 Perhaps something like "highlightedRegion" is clearer?

Reply:
I also find it a bit confusing, but was trying to stay close to the SARIF terms. Since you're having the same issue as me, I think it's a signal that it's worth changing the names. How about `codeSnippet` and `highlightedRegion`?

Reply:
> I also find it a bit confusing, but was trying to stay close to the SARIF terms. Since you're having the same issue as me, I think it's a signal that it's worth changing the names. How about `codeSnippet` and `highlightedRegion`?

Yeah, I think we should either match the SARIF terms exactly (i.e. `contextRegion` and just `region`), or pick something more self-explanatory! I'd vote for your suggestion of `codeSnippet` and `highlightedRegion` 😀

Reply:
Updated!